examples/spring-boot-tomcat/src/main/resources/config/application.yml

```diff
@@ -6,4 +6,4 @@ spring.config.activate.on-profile: local
 armeria:
   ports:
     - port: 8080
-      protocol: HTTP
+      protocols: HTTP
```

Comment:
This field is a list and we can just specify the element without enclosing brackets. e.g. `[HTTP]`
It's because Spring has [relaxed rule](https://docs.spring.io/spring-boot/docs/2.0.1.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding) for the list value.
However, this doesn't apply for Dropwizard.